site/layouts/partials/course_content.html

```diff
@@ -3,10 +3,12 @@
 
 <div class="pb-4 rounded">
   <header>
-    <h1 class="text-uppercase font-weight-bold">{{ $currentPage.Title }}</h1>
-    {{ with $currentPage.Params.course_info.subtitle }}
-    <span class="subtitle">{{ . }}</span>
-    {{ end }}
+    <div class="course-section-title-container">
+      <h1 class="text-uppercase font-weight-bold">{{ $currentPage.Title }}</h1>
+      {{ with $currentPage.Params.course_info.subtitle }}
+      <span class="subtitle">{{ . }}</span>
+      {{ end }}
+    </div>
```

Comment:
was this change accidentally included from https://github.com/mitodl/hugo-course-publisher/pull/364/?

Reply:
@aliceriot I accidentally created this PR from the wrong branch, I'm just going to recreate it as that will be easier.